apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -0,0 +1,67 @@
+# Working with types
```

Comment:
```suggestion
# Working With Types
```

The other main sidebar section titles are title case.

Had to check styleguide to see if 'With' should be capitalized here, but looks like it should: https://en.wikipedia.org/wiki/Title_case

Reply:
https://github.com/blockprotocol/blockprotocol/commit/a99d4f2457ac5c73af8b6b4d164455591eaf0b0a